build-tools/heroku/tailwind/generate-css

```diff
@@ -5,6 +5,11 @@ cd $scriptdir
 
 staticdir=../../../static
 
+if [[ "${1:-}" == "--watch" ]]; then
+  echo "👀 Running Tailwind compilation in watch mode. 👀"
+  npx tailwindcss build -i styles.css -o $staticdir/css/generated.css --watch
+fi
+
 npx tailwindcss build -i styles.css -o $staticdir/css/generated.css
```

Comment:
shouldn't this line then be in an else block?

Reply:
you'll want the minify to run as well I assume?

Reply:
That's not really a problem as the script is run on deploy as well, still minimizing the .css file.